maintainers/scripts/activity/activity.bb

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env bb
+
+;; to get a repl
+;; - Terminal
+;;       $ nix-shell -p babashka curl --run "bb"
+;;       user=> (load-file "activity.bb")
+;;       user=> (pprint (file-query :org-repos {:org "nixos"}))
+;; - IDE support
+;;       $ nix-shell -p babashka curl --run "bb nrepl-server"
+;;   - connect to 127.0.0.1:1667 with emacs/cider or vscode/calva
+
+;; # Configuration
+
+(require '[babashka.curl :as curl]
+         '[babashka.cli :as cli])
+
+(def wd (.getParentFile (fs/file *file*)))
+
+(def graphql-path
+  "Make working directory overridable"
+  (str
+   (or (System/getenv "NIXOS_ACTIVITY_GQL_PATH")
+       wd)))
+
+(def maintainers-path
+  "Make maintainers path overridable"
+  (str
+   (or (System/getenv "NIXOS_MAINTAINERS_PATH")
+       (fs/canonicalize
+        (fs/file wd "../..")))))
+
+;; (println "graphql path:" graphql-path)
+
+(defn get-access-token
+  "GitHub access token lookup"
+  []
+  (or (System/getenv "ACCESS_TOKEN")
```

Comment:
`GITHUB_TOKEN` is generally the environment variable used for those tokens (do a `grep` in nixpkgs you will see there are lots of usages already), so instead of inventing another env variable I think it is better to just reuse that one.

Reply:
done. thanks, I wasn't aware